ee/clickhouse/models/property.py

```diff
@@ -144,17 +181,23 @@ def parse_prop_clauses(
             method = format_static_cohort_query if prop.type == "static-cohort" else format_precalculated_cohort_query
             filter_query, filter_params = method(cohort_id, idx, prepend=prepend, custom_match_field=person_id_joined_alias)  # type: ignore
             if has_person_id_joined:
-                final.append(f" AND {filter_query}")
+                final.append(f" {property_operator} {filter_query}")
             else:
                 # :TODO: (performance) Avoid subqueries whenever possible, use joins instead
                 # :TODO: Use get_team_distinct_ids_query instead when possible instead of GET_TEAM_PERSON_DISTINCT_IDS
                 subquery = GET_DISTINCT_IDS_BY_PERSON_ID_FILTER.format(
                     filters=filter_query, GET_TEAM_PERSON_DISTINCT_IDS=GET_TEAM_PERSON_DISTINCT_IDS
                 )
-                final.append(f"AND {table_name}distinct_id IN ({subquery})")
+                final.append(f" {property_operator} {table_name}distinct_id IN ({subquery})")
             params.update(filter_params)
 
-    return " ".join(final), params
+    # TODO: clean
+    joined = f"AND ({' '.join(final).replace(property_operator, '', 1)})"
```

Comment:
Basically want to AND whatever comes out of this function from now on, so other filters, like team_id & datetimes etc. etc. can be ANDed as expected.

Earlier this wasn't a problem since everything was ANDed anyway